_data/openssl/X509_V_ERR_CERT_NOT_YET_VALID.yml

```diff
@@ -0,0 +1,27 @@
+error-code: X509_V_ERR_CERT_NOT_YET_VALID
+slug: x509-v-err-cert-not-yet-valid
+tags: [ openssl, time ]
+openssl-docs: |
+  The certificate is not yet valid: the notBefore date is after the current time.
+gnutls-code: GNUTLS_CERT_NOT_ACTIVATED
+gnutls-docs: |
+  The certificate is not yet activated.
+botan-code: CERT_NOT_YET_VALID
+botan-docs: |
+  Certificate is not yet valid
+mbed-code: MBEDTLS_X509_BADCERT_FUTURE
+mbed-docs: |
+  The certificate validity starts in the future
+weight: 101
+verify-cert: X509_V_ERR_CERT_NOT_YET_VALID
+verify-command: |
```

Comment:
Eventually, we might be adding more than one chain to each error.
I just realized that there might be an issue with that here. The verify command might not be the same for all example certs within one error (even when we get to validate a different way, say with our own TLS clients).

Wouldn't it make sense to store the verify command for every library with the example chain and only store the expected message here?

However, that would mean that the tests would run not through all errors in `_data/*/*.yml`, but through all example certificates and only look for the expected message here.
A longer discussion is probably needed about this.

Reply:
Yes, I was also thinking about that. My plan was:
* To eventually change the verify-cert to a list of cert references instead of a single one.
* If different verify commands would be necessary, to have tuples (cert-reference; verify cmd) in the example certificate.
This way the testing would still iterate through errors and we could use the same cert reference for multiple errors (for example with different CLI options).